Tools/peg_generator/scripts/benchmark.py

```diff
@@ -43,16 +47,17 @@
 
 def benchmark(func):
     def wrapper(*args):
+        print(f"{func.__name__}")
```

Comment:
I just ran into a situation where this was quite confusing. `make time_compile` prints `time_compile` here. But `make time_stdlib_compile` *also* prints `time_compile` here. Maybe the printing should happen at an earlier point and it should print more info about the parameters to the program?